Dockerfile

```diff
@@ -1,17 +1,27 @@
-FROM debian:8
+FROM node:carbon-alpine as build-frontend
+WORKDIR /frontend
+COPY frontend/package.json ./
+COPY frontend/yarn.lock ./
+RUN yarn --production
+COPY frontend .
+RUN npm run build
 
-ADD backend.tar.gz /www
+FROM golang:1.9-alpine as build-backend
+WORKDIR /go/src/github.com/HackGT/SponsorshipPortal/backend
+RUN apk update && apk add git
+RUN go get -u github.com/golang/dep/cmd/dep
+RUN go get -u github.com/revel/cmd/revel
```

Comment:
maybe make a script to people can test it outside of docker

Reply:
That doesn't feel DRY because the script would span both the `build-frontend` and `build-backend` stage and that makes it hard not to make. `dep` and `revel` (soon to be removed) are system dependencies that users will need installed already. The only thing that this is doing is copying the contents of the `build` directory in the `build-frontend` stage into `app/public` (needs to be changed to `public`) directory of `build-backend`

Normal development does not need to do this